Flash Chat iOS13/Controllers/ThreadListViewController.swift

```diff
@@ -17,11 +17,30 @@ class ThreadListViewController: UITableViewController {
 
 	override func viewDidLoad() {
 		super.viewDidLoad()
+		navigationItem.hidesBackButton = true
 		loadThreads()
 	}
 
 	func loadThreads() {
-
+		database.collection(Constants.FStore.usersCollectionName)
+			.whereField(Constants.FStore.emailField, isEqualTo: (Auth.auth().currentUser?.email)!)
+			.getDocuments { [self] snapshot, error in
+				if let error = error {
+					AppDelegate.showError(error, inViewController: self)
+					return
+				} else {
+					if (snapshot?.documents.isEmpty)! {
+						let data: [String : Any] = [
+							Constants.FStore.emailField : (Auth.auth().currentUser?.email)!
+						]
+						database.collection(Constants.FStore.usersCollectionName).addDocument(data: data) { [self] error in
+							if let error = error {
+								AppDelegate.showError(error, inViewController: self)
+							}
+						}
+					}
+				}
+			}
```

Comment:
⚒️ 
I don't think we want to "fetch all users" when loading threads.

```suggestion
```

Reply:
Meant to put the array comment here...